enocean/README.md

```diff
@@ -1,23 +1,63 @@
 # EnOcean
 
-This plugin is still under development. If you have special hardware not supported yet please feel free to improve and contribute!
+## Description
+This plugin allows you to interact with enocean devices via SmarthomeNG and SmartVisu.
+
+This plugin is still under development.
+
+## Support
+If you have special hardware not supported yet please feel free to improve and contribute!
+
+## Version / Change History
+Version: 1.6
+
+Change History: actually not maintained.
 
 ## Requirements
+For use of this plugin you need an enocean radio transceiver module like:
+- Fam4Pi
+- USB 300
+- ENOCEAN PI 868 Funk Modul
+- etc.
 
 ## Configuration
 
-### plugin.conf
+### plugin.yaml / plugin.conf (deprecated)
+Add the following lines to your `plugin.yaml` or `plugin.conf`.
+
+#### Parameters
+##### serialport
+You have to specify the `serialport` to your port name of your enocean-adpater.
+Creating **udev-rules** for the enocean-adapter is recommend, when using different uart devices.
+
+##### tx_id
+The specification of the enocean `tx_id` is optional **but** mandatory for sending control commands from the enocean-adapter to an enocean device. 
+It is defined as a 8-digit hex value.
 
-Add the following lines to your ``plugin.conf`` and just adapt the serial port to your port name of your enocean-adpater.
-A udev-rules for the enocean-adapter is recommend, when using different uart devices.
-The specification of the enocean tx_id is optional but mandatory for sending control commands from the stick to a device. 
-It is defined as a 8 digit hex value.
+When controlling multiple devices, it is recommended to use the enocean-adapter's Base-ID (not Unique-ID or Chip-ID) as transmitting ID.
+For further information regarding the difference between Base-ID and Chip-ID, see
+[Knowledge Base](https://www.enocean.com/en/knowledge-base-doku/enoceansystemspecification%3Aissue%3Awhat_is_a_base_id/)
 
-When controlling multiple devices, it is recommended to use the stick's BaseID (not ChipID) as transmitting ID.
-For further information regarding the difference between BaseID and ChipID, see [Knowledge Base](https://www.enocean.com/en/knowledge-base-doku/enoceansystemspecification%3Aissue%3Awhat_is_a_base_id/)
+With the specification of the Base-ID, 128 different transmit ID's are available, ranging between Base-ID and Base-ID + 127.
 
-With the specification of the BaseID, 128 different transmit IDs are available, ranging between BaseID and BaseID + 127.
+##### How-To Get the ID of an EnOcean device
```

Comment:
What is the correct notation for enocean? EnOcean, ENOCEAN, enocean, ...? We should stick to one notation and use it everywhere in the documentation. Only a minor thing, but makes the documentation looking cleaner.